apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx

```diff
@@ -161,6 +164,20 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
         setFilterSelectedExternalClaims(tempFilterSelectedExternalClaims);
     }, [ claimConfigurations ]);
 
+    useEffect(() => {
+        if (unfilteredExternalClaimsGroupedByScopes.length != 0) {
+            const initialSelectedScopes = [ "openid" ];
```

Comment:
convert this to a constant

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3640/commits/1eea89c4b4ff909d642ad66bb648e32a18e60490